source/adminguide/host_and_storage_tags.rst

```diff
@@ -31,30 +31,39 @@ There are two types of host tags:
 To explain the behavior of host tags, some examples will be demonstrated with two hosts (Host1 and Host2):
 
 #. Tag setup:
+
     * Host1: h1
     * Host2: h2
     * Offering: h1
+
     When a VM is created with the offering, the deployment will be carried out on Host1, as it is the one that has the tag compatible with the offering.
```

Comment:
```suggestion
When a VM is created with the offering, the deployment will be carried out on Host1, as it is the one that has the tag compatible with the offering.
```

Reply:
It is a paragraph under "#. Tag Setup", as I see it. and not related to the specific last of the tags mentioned above as it seemed to be before. The indentation is a bit overdone. Thanks anyway, I'm not going to stop these improvements over this.